### PR TITLE
EZP-30386 Custom buttons in Online Editor - 2.5.1

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-30386
| Versions      | master, 2.5.1

### CODE REVIEW

File: docs/guide/extending_online_editor.md [hunk 1]
```diff
@@ -275,3 +274,22 @@ The name of a plugin needs to be the same as the one passed to `CKEDITOR.plugins
...
+ezpublish:
+    ezrichtext:
+        alloy_editor:
+            extra_buttons:
+                - paragraph: [buttonName1, buttonName2]
+                - embed: [buttonName1]
+```
+
+Under `extra_buttons` (line 4) you need to specify to what toolbar you want to add a new button e.g. `paragraph`.
+Next to that toolbar, add a table with names of custom buttons that you want to install e.g. `[buttonName1, buttonName2]`.
```
Comment (dew326) [inline docs/guide/extending_online_editor.md]:
```suggestion
Next to that toolbar, add an array with names of custom buttons that you want to install e.g. `[buttonName1, buttonName2]`.
```

File: docs/guide/extending_online_editor.md [hunk 2]
```diff
@@ -275,3 +274,22 @@ The name of a plugin needs to be the same as the one passed to `CKEDITOR.plugins
...
+```yml hl_lines="4"
+ezpublish:
+    ezrichtext:
+        alloy_editor:
+            extra_buttons:
+                - paragraph: [buttonName1, buttonName2]
+                - embed: [buttonName1]
+```
+
+Under `extra_buttons` (line 4) you need to specify to what toolbar you want to add a new button e.g. `paragraph`.
```
Comment (michalmilc90) [inline docs/guide/extending_online_editor.md]:
```suggestion
Under `extra_buttons` (line 4) specify to what toolbar you want to add a new button e.g. `paragraph`.
```

File: docs/guide/extending_online_editor.md [hunk 3]
```diff
@@ -275,3 +274,22 @@ The name of a plugin needs to be the same as the one passed to `CKEDITOR.plugins
...
+Next to that toolbar, add an array with names of custom buttons that you want to install e.g. `[buttonName1, buttonName2]`.
+
+All new buttons should also be added to AlloyEditor under the same name that was provided in the configuration file.
```
Comment (michalmilc90) [inline docs/guide/extending_online_editor.md]:
```suggestion
All new buttons should also be added to AlloyEditor under the same name that in the configuration file.
```

**Comment by MagdalenaZuba** [comment]
OK, good to know, we will change it in 3.0, until then it should be treated as a normal feature.

File: docs/guide/extending_online_editor.md [hunk 4]
```diff
@@ -275,3 +274,21 @@ The name of a plugin needs to be the same as the one passed to `CKEDITOR.plugins
...
+
+## Buttons configuration
+
+Custom buttons can be added to your installation with the following configuration:
+
+```yml hl_lines="4"
+ezrichtext:
+    alloy_editor:
+        extra_buttons:
+            - paragraph: [buttonName1, buttonName2]
```
Comment (micszo) [inline docs/guide/extending_online_editor.md]: The hyphens should be removed. 😉

**Pull request merged**